api-specificatie/openapi.yaml

```diff
@@ -1164,9 +1132,7 @@ components:
           type: boolean
         geometrie:
           type: boolean
-        hoofdadresIdentificatie:
-          type: boolean
-        nevenadresIdentificaties:
+        nummeraanduidingIdentificaties:
```

Comment:
Hoe weet een gebruiker straks nog of het onderzoek een hoofd- of nevenadres betreft?